NotePad/res/layout/note_editor.xml

```diff
@@ -16,6 +16,14 @@
  * limitations under the License.
  -->
 
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
It would be nice to have a screenshot in the pull request to see how it looks like